pkg/thanos/statefulset.go

```diff
@@ -254,6 +257,15 @@ func makeStatefulSetSpec(tr *monitoringv1.ThanosRuler, config Config, ruleConfig
 		trCLIArgs = append(trCLIArgs, monitoringv1.Argument{Name: "alert.relabel-config-file", Value: fullPath})
 	}
 
+	if tr.Spec.Web != nil {
```

Comment:
we need a version check here (web config requires Thanos v0.21.0 at least). If Thanos >= v0.21.0, the operator should always mount the TLS assets volume in the thanos container even if empty.

Reply:
so the if condition should be like this?
`version.GTE(semver.MustParse("0.21.0"))`

Reply:
correct